### ISSUE TITLE
Documentation of missing functions listed in tests

### ISSUE THREAD

**Issue by voronoipotato** [krauthaufen/FShade#17]
It would have been really helpful when I was messing with this a few months ago and couldn't figure out how to do "distance" :). Thanks for all the hard work on building this by the way, it's awesome!

**Comment by krauthaufen** [comment]
Hey, glad that you like it. I'm aware that most of it is not really documented at all and we'd definitely like to improve that, but what exactly do you mean by missing functions list? There recently has been a major overhaul of aardvark.base's matrix/vector things which has certainly caused several missing functions.
I think it would be great to have some sort of reference for builtin functions, etc.

**Comment by voronoipotato** [comment]
yeah that's what I mean, it's not clear what functions are available for example in glsl. For example, in glsl there's a distance function, it's unclear how to represent that in a FShade shader.

**Comment by krauthaufen** [comment]
Hey, I totally see the issue, Would be really nice to have documentation on that, maybe we could auto-generate it partly by using FShade's intrinsic function table somehow.
